src/main/groovy/com/twcable/grabbit/jcr/JcrUtil.groovy

```diff
@@ -164,4 +166,30 @@ public class JcrUtil {
         return withResourceResolver(resolverFactory, "admin", closure)
     }
 
+    /**
+     * @return true if the node at the path being written is being written to an existing node
+     */
+    static boolean writingToExistingNode(@Nonnull final String pathBeingWritten, @Nonnull final Session session) {
```

Comment:
I am also having tough time thinking about a name for this.. :P writesToExistingNode or what you currently have should be fine.. 👍